includes/oauth/class-google-login.php

```diff
@@ -122,7 +122,8 @@ public static function oauth_callback() {
 		}
 
 		if ( ! wp_verify_nonce( sanitize_text_field( $_GET[ self::AUTH_CALLBACK ] ), self::AUTH_CALLBACK ) ) {
-			self::handle_error( __( 'Nonce verification failed.', 'newspack-plugin' ) );
+			/* translators: %s is a unique user id */
+			sprintf( __( 'Nonce verification failed for id: %s', 'newspack-plugin' ), OAuth::get_unique_id() );
```

Comment:
Looks like you accidentally removed the call to `self::handle_error

Reply:
Good catch! Added back in 2b3a8b9e1cda7d6898c293e195c9d804259d5e87